### DISCUSSION TITLE
Screen is returning 404, but it apparently the Model is found

### DISCUSSION THREAD

**Discussion by ppalmeida** [orchidsoftware/platform#2278] [Q&A]
Hey, folks.

I started with Orchid recently. After installing and making a few changes, I decided to try the "CRUD" package. It's installed and it looks ok. But now the "Users" Screen (the **regular Screen one, not the CRUD User Resource**) started to throw 404 in the edit screen. It can list the `users`, but when I click `Edit` in the data table, it shows the default Laravel's 404 page.

I was digging and this is what I found after a few steps: in the file 
 [ScreenDependencyResolver.php line 73](https://github.com/orchidsoftware/platform/blob/master/src/Screen/Resolvers/ScreenDependencyResolver.php#L73) there is a `throw_if` and performing a `dd()` right before that, this is what I can see:
 
* `$instance` variable has a empty (no attributes fulfiled) `\App\Models\User` instance.
* `$value` variable **has the exact `\App\Models\User` I am trying to edit,  with all attributes correctly fulfilled (id, name, email, etc, it's all there)**
* `$model` variable is `NULL`: which means `$instance->resolveRouteBinding($value)` is not doing correctly it's job. Does this `resolveRouteBiding` method come from Laravel?

Then, because `$model` is `NULL` and `$parameter->isDefaultValueAvailable()` returns `FALSE` the `ModelNotFoundException` is thrown. Where should I look or change to make the `Edit User Screen` work again?

By the way, this is the route signature I have in `~routes/platform.php` file:

```php 
// Platform > System > Users
Route::screen('users/{user}/edit', UserEditScreen::class)
    ->name('platform.systems.users.edit')
    ->breadcrumbs(function (Trail $trail, $user) {
        return $trail
            ->parent('platform.systems.users')
            ->push(__('User'), route('platform.systems.users.edit', $user));
    });
```

Sorry if this sounds a bit confusing, I am just trying to go deeper, understand how the lib works, and discover why it returns 404, although the `$value` variable is correctly set with the entity I want to edit. 

Maybe I shouldn't use the CRUD package and regular Orchid Screens together in the same application? But seeing all the flexibility and high-quality code of Orchid, I don't think it would be the case.

**Comment by tabuna** [comment]
Hi @ppalmeida, I use both packages in my daily life and didn't face this problem. 
To help you, I need to know how to reproduce this behavior.
